android/src/com/mapswithme/maps/widget/placepage/PlacePhoneAdapter.java

```diff
@@ -0,0 +1,122 @@
+package com.mapswithme.maps.widget.placepage;
+
+import android.content.Context;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.PopupMenu;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import com.mapswithme.maps.R;
+import com.mapswithme.util.Utils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class PlacePhoneAdapter extends RecyclerView.Adapter<PlacePhoneAdapter.ViewHolder>
+{
+  public static String TAG = "PlacePhoneAdapter";
+
+  private List<String> mPhoneData = Collections.emptyList();
+
+  public PlacePhoneAdapter() {}
+
+  public PlacePhoneAdapter(String phones) {
+    refreshPhones(phones);
+  }
+
+  public void refreshPhones(String phones) {
+    Log.w(TAG, "refreshPhones: " + phones);
+    if (TextUtils.isEmpty(phones))
+      return;
+
+    mPhoneData = new ArrayList<>();
+    for (String p : phones.split(";"))
+    {
+      p = p.trim();
+      if (TextUtils.isEmpty(p)) continue;
+      Log.w(TAG, "adding phone: "+p);
```

Comment:
We try not to pollute logs with debug strings (esp. warning ones).

Reply:
I forgot to remove those logs. It's no use to log this text any more